### ISSUE TITLE
Update contributor page styles to include photos

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/ppa-django#243]
# testing notes
## designs
[desktop](https://zpl.io/agzzQlN) / [tablet](https://zpl.io/a3exLkW) / [mobile](https://zpl.io/29OlLgy)

**Comment by mnaydan** [comment]
@thatbudakguy right now portrait photos aren't being formatted in the way Gissoo designed. See Roland's example, for instance below. Is this something we have control over in Wagtail? 
Gissoo's design:
<img width="278" alt="screen shot 2019-02-27 at 10 01 37 am" src="https://user-images.githubusercontent.com/22945600/53499786-13d8b800-3a77-11e9-8879-955ab25a7042.png">
Test:
<img width="276" alt="screen shot 2019-02-27 at 10 01 44 am" src="https://user-images.githubusercontent.com/22945600/53499790-163b1200-3a77-11e9-9149-29f5d2aac0c1.png">

**Comment by wafaisfahany** [comment]
- Photos not as Gissoo's design (do they still need to be?) 
-  For Desktop: Photos (i.e tiles) not spaced out as in Gissoo's design - items too close together
- "center for digital Humanities" text warped in most legends

**Comment by thatbudakguy** [comment]
I changed the profile picture display slightly so that images with the gray borders added (like Meredith's) should display correctly. However, the people whose images have the wrong aspect ratio (they're tall instead of long) will still look bad - I think you might want to quickly crop them so they're wider than they are tall. @bwhicks is doing some work so that this cropping would automatically focus on the faces, but there are some problems with it right now. I think it's probably better to just edit the images first so that wagtail can do a better job cropping them.

I spaced the profile cards out a little bit more on desktop also. I think the wrapping of the text in the job titles is probably OK since the font size looks to be the correct one but would defer to @mnaydan .

**Comment by rlskoeser** [comment]
@thatbudakguy did you try the `-cXX%` option with the wagtail image tag? We don't have automatic focus detection, but I think it can still be manually specified after upload.

**Comment by thatbudakguy** [comment]
I did; unfortunately it seemed to not have any effect. Not sure if openCV needs to be set up in order for it to work.

**Comment by mnaydan** [comment]
@thatbudakguy font, wrapping, spacing look fine to me.
I don't totally understand what's going on with the images, I'm sorry--so the grey border isn't something that Wagtail will add? It's something that would have to be added in photo editing outside of Wagtail? I edited Roland's to be wider than it is tall and it looks slightly better. Will editing the photos in this way be our solution to this problem, so the tall photos won't end up looking like GD's design?  
<img width="272" alt="screen shot 2019-02-28 at 2 44 07 pm" src="https://user-images.githubusercontent.com/22945600/53593714-58438100-3b67-11e9-88f7-c050bdb350ac.png">

**Comment by thatbudakguy** [comment]
right; wagtail does not generate a background. we're asking wagtail to take an image and always fill the entire card, which means it will crop the photo however it has to so that it fills the whole space. if we want to show more of the subject, we have to add more area to the photo so that wagtail has more to crop.

**Comment by thatbudakguy** [comment]
to expand on this - it's possible that we could crop the image so that it doesn't totally fill the card, and then have the gray as a background underneath it (this is how the placeholder image works when lazy loading). however, the problem with this is that we'd still want it to fill the whole card when possible, and wagtail doesn't know the difference - either it's cropping the whole thing to the card, or it's cropping it to some other dimensions, but it's gotta be the same for every image.

**Comment by mnaydan** [comment]
I managed to make the pictures work by cropping them in preview beforehand to make them more wide than they are tall. It looks fine on desktop and tablet, but the image block is narrower for the mobile specs, such that the pictures are cropped too much. Could @gissoo advise here? Would it be possible to make the cards the same size as on desktop and tablet for mobile? Closing and tracking the mobile issue on #267.
